lib/generators/blacklight/templates/catalog_controller.rb

```diff
@@ -53,9 +53,14 @@ class <%= controller_name.classify %>Controller < ApplicationController
     #
     # :show may be set to false if you don't want the facet to be drawn in the 
     # facet bar
+    
+    # set :index_pagination to true if you want the pop-up window for large facets to have alphabetical index navigation
+    #  (useful when user clicks "more" on a large facet and wants to navigate alphabetically across a large set of results)
+    # if you set :index_pagination to true, you MUST set :index_range to an array of characters that will be used to create the navigation (note: It is case sensitive when searching values)
+    
```

Comment:
The comment says to set `:index_pagination => true` but in the example `:index_pagination` is not set.